tests/redux/reducers/search/searchFiltersReducer-test.js

```diff
@@ -49,17 +49,17 @@ describe('searchFiltersReducer', () => {
     describe('UPDATE_TEXT_SEARCH', () => {
         const action = {
             type: 'UPDATE_TEXT_SEARCH',
-            textInput: "testing"
+            textInput: 'testing'
         };
 
-        const keyword = "testing";
+        const keyword = 'testing';
 
 
         it('should add the provided keyword if it does not currently exist in the filter', () => {
             const updatedState = searchFiltersReducer(undefined, action);
 
             expect(updatedState.keyword).toEqual(
-                new OrderedMap({ "testing": keyword })
+                new OrderedMap({ 'testing': keyword })
```

Comment:
You don't need quotes around property names unless there are special characters in it.